### PR TITLE
Pruzhinina

### PR DESCRIPTION
**Pull request by arinapruzhinina**

### CODE REVIEW

**Comment by siriusdevs** [comment]
your script doesnt work, did you even check it?

File: hw_quiz/script.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+export PG_PASSWORD=123
```
Comment (siriusdevs) [inline hw_quiz/script.sh]: are you sure?

**Review by siriusdevs — CHANGES_REQUESTED** [review]

File: hw_quiz/script.sh (hunk 1 above)
Comment (arinapruzhinina) [reply hw_quiz/script.sh]: fixed

**Comment by siriusdevs** [comment]
1) if i input the wrong type of answer (for example, text instead of a number), you tell me its wrong. then i proceed with answering "нет" on a question "Хотите сыграть ещё раз?". Nevertheless, the game continues..
2) if i input the wrong answer (for example, 1 instead of 2), this question will reappear until i will eventually give the right answer.

**Pull request merged**